challenge.rb

```diff
@@ -1,12 +1,17 @@
 # Assessment :: Ruby Basics
 
+require 'pry'
 
 # Question 1
 # Complete the method `is_of_age?`. It should accept either strings or numbers in its arguments,
 # and should return either `true` or `false` based on whether `age` is less than `minimum_age`.
 
 def is_of_age? age, minimum_age
-  # Replace this comment with your code.
+  if (age.to_s >= minimum_age.to_s) || (age.to_i >= minimum_age.to_i)
+    return true
+  else
+    return false
+  end
```

Comment:
This is correct, but could have been written more tersely.
```
def is_of_age? age, minimum_age
   age >= minimum_age
end
```
Do you see how these work out to be the same?